tests/support/Traits/SqlMockTrait.php

```diff
@@ -145,6 +145,11 @@ protected function mockGetSearchResultWithTrashed(): void
         $this->mockSelect('select * from `test_models` order by `id` asc limit 15 offset 0', [], [], $pdo);
     }
 
+    protected function mockGetCountOfUsersExceptAuthorized(int $count): void
+    {
+        $this->mockSelect("select count(*) as entities_count from users where id <> 1 and email = 'mail@mail.com';", [], [['entities_count' => $count]]);
```

Comment:
```suggestion
        $this->mockSelect("select count(*) as entities_count from users where id <> 1 and email = 'mail@mail.com';", [], [
            ['entities_count' => $count]
        ]);
```

Reply:
@DenTray I rearranged code in a whole file